code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -200,6 +200,14 @@
 		bio = ARMOR_BIO_SHIELDED,
 		rad = ARMOR_RAD_SHIELDED
 	)
+	slowdown = 0
+	offline_slowdown = 2
+	offline_vision_restriction = 0
+	siemens_coefficient = 0.5
```

Comment:
Parent class `/obj/item/rig` sets it as 0.35, is it intentional for this to have a worse siemens coefficient?